sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/administration/package-info.java

```diff
@@ -2,9 +2,139 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing classes for creating a
- * {@link com.azure.ai.formrecognizer.documentanalysis.administration.DocumentModelAdministrationAsyncClient} and
- * {@link com.azure.ai.formrecognizer.documentanalysis.administration.DocumentModelAdministrationClient} to perform operations on
- * Azure Form Recognizer document model management operations.
+ * <p>The Azure Form Recognizer library provides
```

Comment:
The review comments for file `src/main/java/com/azure/ai/formrecognizer/documentanalysis/package-info.java`
apply here as well.